app/controllers/admin/registries_controller.rb

```diff
@@ -60,8 +79,30 @@ def edit
   # given registry. This might change in the future.
   def update
     @registry = Registry.find(params[:id])
-    @registry.update_attributes(update_params)
-    redirect_to admin_registries_path, notice: "Registry updated successfully!"
+    @registry.assign_attributes(update_params)
+    @can_change_hostname = !Repository.any?
+
+    # Check the reachability of the registry.
+    unless params[:force]
```

Comment:
There is no `force` parameter on the edit form, so this shouldn't apply here.

Reply:
We didn't but when I started this I saw the opportunity to improve the edit form experience the same way I did for the create form. So we have validation on the hostname value too. What do you think?

Reply:
I think it's alright, but the `force` check point was meant more like "I can't connect to my registry now and I don't know why, but I want to test Portus out anyways". Kind of bad but it might come in handy when the registry is not up when you are configuring Portus but that's really the config for it.

So, with this in mind, I don't think it makes much sense to have this same checkbox on the edit form (it does make a lot of sense to check whether is reachable or not of course, so I appreciate your changes on this regard to the `update` method 👍).

Reply:
It makes sense. So no force on edit but validation, right? I'll change that.